api/v1alpha/client_example.go

```diff
@@ -29,8 +30,11 @@ import (
 
 func getLogsWithoutFollow(c v1alpha.PublicAPIClient, p *v1alpha.Pod) {
 	logsResp, err := c.GetLogs(context.Background(), &v1alpha.GetLogsRequest{
-		PodId:  p.Id,
-		Follow: false,
+		PodId:     p.Id,
+		Follow:    false,
+		AppName:   p.Apps[0].Name,
```

Comment:
we maintain the invariant of `len(p.Apps) > 0` somewhere? otherwise this will panic.